posthog_test.go

```diff
@@ -741,21 +742,9 @@ func TestFeatureFlagsWithNoPersonalApiKey(t *testing.T) {
 	})
 	defer client.Close()
 
-	receivedErrors := [4]error{}
+	receivedErrors := [2]error{}
 	receivedErrors[0] = client.ReloadFeatureFlags()
-	_, receivedErrors[1] = client.IsFeatureEnabled(
-		FeatureFlagPayload{
-			Key:        "some key",
-			DistinctId: "some id",
-		},
-	)
-	_, receivedErrors[2] = client.GetFeatureFlag(
-		FeatureFlagPayload{
-			Key:        "some key",
-			DistinctId: "some id",
-		},
-	)
-	_, receivedErrors[3] = client.GetFeatureFlags()
+	_, receivedErrors[1] = client.GetFeatureFlags()
```

Comment:
we only have 2 methods that error on a missing PersonalApiKey, and neither of them are hot paths (i.e. users don't need them to use our feature flags).